src/main/resources/static/mini-manage-api-stubs.yml

```diff
@@ -376,18 +376,52 @@ components:
           format: uuid
         name:
           type: string
+          example: Hope House
         apCode:
           type: string
-        probationRegionId:
-          type: string
+          example: NEHOPE1
         postcode:
           type: string
+          example: LS1 3AD
         bedCount:
           type: integer
-        apAreaId:
-          type: string
+          example: 22
+        probationRegion:
+          allOf:
```

Comment:
Minor question, is there any reason for using `allOf` over:

```
        probationRegion:
          $ref: '#/components/schemas/ProbationRegion'
```

Reply:
thanks Joseph, good point! I'll tweak that...